tests/backends/sqlalchemy/test_objects.py

```diff
@@ -26,6 +26,14 @@ def test_activity_has_facts(self, alchemy_store, alchemy_fact_factory):
         assert activity.facts
 
 
+class TestAlchemyTag(object):
+    """Make sure our custom methods behave properly."""
+    def test_as_hamster(self, alchemy_store, alchemy_tag):
+        """Make sure that conversion into a ``hamster_lib.Activity```works as expected."""
```

Comment:
Should be `hamster_lib.objects.Tag` and the third backtick should be replaced with a space.